krysmathis/NSSEscape
# added 2nd page of alumni into database, starting with Day cohort 9.

added 2nd page of alumni into database, starting with Day cohort 9, Refactored EnterAlumnusMenu.cs to allow user to quit from either prompt

To Test:
1. Git Fetch branch
1. Git checkout into branch
1. Run `dotnet run`
1. Choose menu option 3 to add an alumni
1. Type "quit" and make sure you are sent back to the first menu screen
1. Choose menu option 3 again
1. choose a cohort, say Day1
1. Type "quit" when prompted to add a student name and make sure you are send back to the first menu screen
1. High five if it all works.
